config.json

```diff
@@ -517,6 +523,12 @@
             "ContainerName": "default"
           }
         },
+        "nodeReuse":{
```

Comment:
Yeah :( that is the reason.
I thought about doing that and then in every .cmd and .sh file pass the correspondent MsbuildParameter setting, but was not sure on which solution would be cleaner.

I opted for this one as the difference was only the nodeReuse parameter. But I'm open to change it,